tests/blockchain/test_org_hierarchy.py

```diff
@@ -373,7 +373,7 @@ def test_03_confirm_update_manager_proposal(self):
                 user_id=self.key2a.public_key,
                 manager_id=self.key_manager.public_key,
             )[0]["status"],
-            "COMMITTED",
+            "INVALID",
```

Comment:
Shouldn't these still come out as COMMITTED with just name changes?  The result of the test should not be different if it is just name changes.

Reply:
ahhh I see. Sorry, I was just seeing why the build was failing, but now I see that these are all because the tests all came back as "INVALID" instead of "COMMITTED".

Reply:
ahhh I see. Sorry, I was just seeing why the build was failing, but now I see that these are all because the tests all came back as "INVALID" instead of "COMMITTED".

Reply:
Sweet.  Can you switch this back then?